desktop/core/src/desktop/settings.py

```diff
@@ -95,7 +95,7 @@
   ('ja', _('Japanese')),
   ('ko', _('Korean')),
   ('pt', _('Portuguese')),
-  ('pt_BR', _('Brazilian Portuguese')),
+  ('pt-br', _('Brazilian Portuguese')),
   ('zh-CN', _('Simplified Chinese')),
```

Comment:
Can't see it in https://github.com/django/django/blob/master/django/conf/global_settings.py ?

Should we also rename all the folders in the `locale` directories? https://github.com/cloudera/hue/tree/master/apps/about/src/about/locale

How to test that `make locales` still works:
Also update the docs?
https://docs.gethue.com/developer/development/#internationalization

Reply:
1-> https://github.com/django/django/blob/master/django/conf/global_settings.py#L124

2-> https://docs.djangoproject.com/en/3.1/topics/i18n/
not need to change locale name i think,  ss
<img width="1006" alt="Screenshot 2021-02-17 at 1 15 16 PM" src="https://user-images.githubusercontent.com/36241930/108171910-3069d100-7122-11eb-90dc-78777d44b549.png">

3-> after running make locales in a local branch. (around 151 files get modified automatically) and then it gives an error
<img width="1792" alt="Screenshot 2021-02-17 at 1 52 55 PM" src="https://user-images.githubusercontent.com/36241930/108176006-72e1dc80-7127-11eb-990b-d278f46533d2.png">

Reply:
We can track the `make locales` in a follow-up. Here you can see that something is not compatible anymore in macros.mako. I bet opening up the Dashboard does not work anymore (can do in follow-up up).

Reply:
okay